microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -620,4 +620,13 @@ Issued: 19 September 2024
 
 {product-title} release 4.14.37 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:6690[RHBA-2024:6690] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:6689[RHSA-2024:6689] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-14-38-dp_{context}"]
+=== RHBA-2024:7186 - {microshift-short} 4.14.38 bug fix and security update advisory
+
+Issued: 3 October 2024
+
+{product-title} release 4.14.38 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:7186[RHBA-2024:7186] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:7184[RHSA-2024:7184] advisory.
```

Comment:
These two links are not currently working:
https://access.redhat.com/errata/RHBA-2024:7186[RHBA-2024:7186]
https://access.redhat.com/errata/RHSA-2024:7184[RHSA-2024:7184]
@dfitzmau, WDYT, should I approve this PR as a peer reviewer (the rest looks good to me)?